packages/mobile/src/verify/VerificationInterstitialScreen.tsx

```diff
@@ -43,7 +44,7 @@ class VerificationInterstitialScreen extends React.Component<WithTranslation> {
             <AnimatedCircle />
             <AnimatedCircle />
           </View>
-          <Text style={fontStyles.h1}>{t('interstitial.header')}</Text>
+          <Text style={fontStyles.h2}>{t('interstitial.header')}</Text>
```

Comment:
nit: Define in stylesheet?

Reply:
@tarikbellamine There's not reason to redefine this in the stylesheet if we don't deviate from the predefined value in fontStyles.